obspy/core/tests/data/RESP.XX.NS306..SHZ.GS13.1.2180

```diff
@@ -0,0 +1,86 @@
+#               << RESP_VER	RESP.XX.NS306..SHZ.GS13.1.2180	2011.114	NEW	 >>
+#               << RESP_VER	RESP.XX.NS306..SHZ.GS13.1.2180	2014.070	UPDATED	Updated normalization frequency. >>
+#		<< IRIS SEED Reader, Release 4.8 >>
+#
+#
+#		======== CHANNEL RESPONSE DATA ========
+B050F03     Station:     NS306
+B050F16     Network:     XX
+B052F03     Location:    ??
+B052F04     Channel:     SHZ
+B052F22     Start date:  2006,001,00:00:00.0000
+B052F23     End date:    No Ending Time
+#		=======================================
+#		+               +--------------------------------------------+                +
+#		+               |   Response (Poles & Zeros), NS306 ch SHZ   |                +
+#		+               +--------------------------------------------+                +
+#
+B053F03     Transfer function type:                A [Laplace Transform (Rad/sec)]
+B053F04     Stage sequence number:                 1
+B053F05     Response in units lookup:              M/S - Velocity in Meters Per Second
+B053F06     Response out units lookup:             V - Volts
+B053F07     A0 normalization factor:               1.000000E+00
+B053F08     Normalization frequency:               5.000000E+00
+B053F09     Number of zeroes:                      2
+B053F14     Number of poles:                       2
+#		Complex zeroes:
+#		  i  real          imag          real_error    imag_error
+B053F10-13    0  0.000000e+00  0.000000e+00  0.000000E+00  0.000000E+00
+B053F10-13    1  0.000000e+00  0.000000e+00  0.000000E+00  0.000000E+00
+#		Complex poles:
+#		  i  real          imag          real_error    imag_error
+B053F15-18    0 -4.443000e+00  4.443000e+00  0.000000E+00  0.000000E+00
+B053F15-18    1 -4.443000e+00 -4.443000e+00  0.000000E+00  0.000000E+00
+#		
+#		+                  +---------------------------------------+                  +
+#		+                  |       Channel Gain, NS306 ch SHZ      |                  +
+#		+                  +---------------------------------------+                  +
+#		
+B058F03     Stage sequence number:                 1
+B058F04     Gain:                                  2.180000e+03
+B058F05     Frequency of gain:                     5.000000e+00 HZ
+B058F06     Number of calibrations:                0
+#		
+#		+               +-------------------------------------------+                 +
+#		+               |   Response (Coefficients), NS306 ch SHZ   |                 +
+#		+               +-------------------------------------------+                 +
+#		
```

Comment:
You will probably have to manually remove all non-PAZ stages from this files to get the test to work.